geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreFactory.scala

```diff
@@ -58,20 +58,20 @@ class CassandraDataStoreFactory extends DataStoreFactorySpi {
     }
     socketOptions.foreach(clusterBuilder.withSocketOptions)
 
-    val user = UserNameParam.lookUp(params).asInstanceOf[String]
-    val password = PasswordParam.lookUp(params).asInstanceOf[String]
+    val user = UserNameParam.lookup(params)
+    val password = PasswordParam.lookup(params)
     if (user != null && password != null) {
       clusterBuilder.withCredentials(user, password)
     }
 
     val cluster = clusterBuilder.build()
     val session = cluster.connect(ks)
-    val catalog = CatalogParam.lookUp(params).asInstanceOf[String]
+    val catalog = CatalogParam.lookup(params)
 
     // not used but required for config inheritance
-    val queryThreads = QueryThreadsParam.lookupWithDefault[Int](params)
-    val queryTimeout = GeoMesaDataStoreFactory.queryTimeout(params)
-    val looseBBox = LooseBBoxParam.lookupWithDefault[Boolean](params)
+    val queryThreads = QueryThreadsParam.lookup(params)
+    val queryTimeout = QueryTimeoutParam.lookupOpt(params).map(_ * 1000L)
```

Comment:
Ok, scaling from seconds to millis here worries me a little.  Is there a good way to indicate what the scale is?

Reply:
I believe it says in the parameter description that it's in seconds. Previously this was done in `GeoMesaDataStoreFactory.queryTimeout(params)`. I think we have a ticket to standardize parameter units - i've been trying to use scala Durations for most things now, which let you do use a variety of formats (1s, 10 seconds, 5minutes, etc). I change this as a part of that ticket if you want

Reply:
we have a ticket for that to standardize everywhere which we should do for 1.4.0 https://geomesa.atlassian.net/browse/GEOMESA-1009 does that jive?

Reply:
Standardizing in another ticket or year works for me!